<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeometryEditor</class>
 <widget class="QWidget" name="GeometryEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>100</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_position">
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit_position_x"/>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="lineEdit_position_y"/>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="lineEdit_position_z"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_rotation">
     <property name="text">
      <string>Rotation</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_rotation_x"/>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineEdit_rotation_y"/>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="lineEdit_rotation_z"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_scale">
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_scale_x"/>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="lineEdit_scale_y"/>
   </item>
   <item row="2" column="3">
    <widget class="QLineEdit" name="lineEdit_scale_z"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
